### ISSUE TITLE
is this project dead?

### ISSUE THREAD

**Issue by indprtech** [halfmoonui/halfmoon#134]
Just wondering what happend to this project last commit 3-2y ago

**Comment by JTorresConsulta** [comment]
Last year forgot pay the domain and was down a few weeks  ^^

I am praying for a .zip with the documentation ot prevent the domain extintion

**Comment by vedranvuk** [comment]
> Last year forgot pay the domain and was down a few weeks ^^
> 
> I am praying for a .zip with the documentation ot prevent the domain extintion

`wget --recursive --convert-links --user-agent=Mozilla https://www.gethalfmoon.com` to get the docs locally

**Comment by Supernova3339** [comment]
docs are written in django

**Comment by indprtech** [comment]
> > Last year forgot pay the domain and was down a few weeks ^^
> > I am praying for a .zip with the documentation ot prevent the domain extintion
> 
> `wget --recursive --convert-links --user-agent=Mozilla https://www.gethalfmoon.com` to get the docs locally

Copying Docs Locally works then doing serve from npm makes it alive.

**Comment by mrintangble** [comment]
Is there another place/team/developer maintaining this project?

**Comment by Supernova3339** [comment]
> Is there another place/team/developer maintaining this project?

Used to be, not anymore

**Comment by Art9681** [comment]
This is a lovely framework and as of last night my pick for a stealth project. I am willing to sponsor and/or take over this project along with some interested contributors to keep it going. I'm willing to fund some bounties for new features for this framework to remain competitive. I am also interested in finding out what the original development team is up to. Please contact me if you are a person that can arrange this. It would be a shame if this was abandoned.

**Comment by Supernova3339** [comment]
You'll have to add new community stuff and make your own repo

**Comment by larsjarred9** [comment]
Just use bootstrap dark at this point ngtl this project has no real-use anymore.

**Comment by Supernova3339** [comment]
> Just use bootstrap dark at this point ngtl this project has no real-use anymore.

Semantic UI hasn't been updated for 5 years ( as of now). So, the community made Fomantic UI.

Why can't we as a community just fork **halfmoonui** and keep adding awesomeness? This library has a lot of potential and I wouldn't want to see it die out.

**Comment by vinidiktov** [comment]
> > Just use bootstrap dark at this point ngtl this project has no real-use anymore.
> 
> Semantic UI hasn't been updated for 5 years ( as of now). So, the community made Fomantic UI.
> 
> Why can't we as a community just fork **halfmoonui** and keep adding awesomeness? This library has a lot of potential and I wouldn't want to see it die out.

There's https://github.com/ksenginew/fullmoon but the project has been archived.

**Comment by Supernova3339** [comment]
> > > Just use bootstrap dark at this point ngtl this project has no real-use anymore.
> > 
> > 
> > Semantic UI hasn't been updated for 5 years ( as of now). So, the community made Fomantic UI.
> > Why can't we as a community just fork **halfmoonui** and keep adding awesomeness? This library has a lot of potential and I wouldn't want to see it die out.
> 
> There's https://github.com/ksenginew/fullmoon but the project has been archived.

Why don't we go ahead and fork it, then?

**Comment by FlavioSuarez** [comment]
v2.0 is here!!

**Comment by Art9681** [comment]
Outstanding. Thank you for the notification!

**Comment by halfmoonui** [comment]
Thankfully, this project is not dead! Please read this if you're interested to know more: https://github.com/halfmoonui/halfmoon/discussions/135

**Comment by jimkring** [comment]
@halfmoonui One thing that I would LOVE ❤️ is to have a few simple website templates to get me started. For example, could we create an `examples` repository that shows how to use all the pieces together in some common ways?  Right now, the [Starter template](https://www.gethalfmoon.com/docs/download/#starter-template) is a (basically) blank page that says "Hello, world!".  As a novice developer, I need some more information to get started.  It might be nice to have a simple flask+jinja app that's styled with halfmoon components.

**Comment by Supernova3339** [comment]
> @halfmoonui One thing that I would LOVE ❤️ is to have a few simple website templates to get me started. For example, could we create an `examples` repository that shows how to use all the pieces together in some common ways? Right now, the [Starter template](https://www.gethalfmoon.com/docs/download/#starter-template) is a (basically) blank page that says "Hello, world!". As a novice developer, I need some more information to get started. It might be nice to have a simple flask+jinja app that's styled with halfmoon components.

I'm working on a VScode extension that will have a template.

**Comment by HarshNarayanJha** [comment]
@Supernova3339, if you need help with the extension, feel free to ask me

**Comment by Supernova3339** [comment]
> @Supernova3339, if you need help with the extension, feel free to ask me

admittedly busy with it right now, converting everything on the documentation to a snippet :)
also adding a few custom ones too, like the theme switcher my [HalfmoonUI Playground](https://github.com/halfmoonui/halfmoon/discussions/137) has.

**Comment by Supernova3339** [comment]
> > @halfmoonui One thing that I would LOVE ❤️ is to have a few simple website templates to get me started. For example, could we create an `examples` repository that shows how to use all the pieces together in some common ways? Right now, the [Starter template](https://www.gethalfmoon.com/docs/download/#starter-template) is a (basically) blank page that says "Hello, world!". As a novice developer, I need some more information to get started. It might be nice to have a simple flask+jinja app that's styled with halfmoon components.
> 
> I'm working on a VScode extension that will have a template.

I haven't finished the extension yet so if I can't get it shipped this week I'll open source my progress so far.

**Comment by FlavioSuarez** [comment]
> > > @halfmoonui One thing that I would LOVE ❤️ is to have a few simple website templates to get me started. For example, could we create an `examples` repository that shows how to use all the pieces together in some common ways? Right now, the [Starter template](https://www.gethalfmoon.com/docs/download/#starter-template) is a (basically) blank page that says "Hello, world!". As a novice developer, I need some more information to get started. It might be nice to have a simple flask+jinja app that's styled with halfmoon components.
> > 
> > 
> > I'm working on a VScode extension that will have a template.
> 
> I haven't finished the extension yet so if I can't get it shipped this week I'll open source my progress so far.
> 
> I have been **SUPER** busy.

Based in your snippets, I could try to create the same for SublimeText...

**Comment by firdaus-aziz** [comment]
> @halfmoonui One thing that I would LOVE ❤️ is to have a few simple website templates to get me started. For example, could we create an `examples` repository that shows how to use all the pieces together in some common ways? Right now, the [Starter template](https://www.gethalfmoon.com/docs/download/#starter-template) is a (basically) blank page that says "Hello, world!". As a novice developer, I need some more information to get started. It might be nice to have a simple flask+jinja app that's styled with halfmoon components.

I agree. At least on v1 we had this documentation https://halfmoonui.pythonanywhere.com/docs/page-building/

The layout is superb and general enough that it covers most use cases.

**Comment by Supernova3339** [comment]
> > > @halfmoonui One thing that I would LOVE ❤️ is to have a few simple website templates to get me started. For example, could we create an `examples` repository that shows how to use all the pieces together in some common ways? Right now, the [Starter template](https://www.gethalfmoon.com/docs/download/#starter-template) is a (basically) blank page that says "Hello, world!". As a novice developer, I need some more information to get started. It might be nice to have a simple flask+jinja app that's styled with halfmoon components.
> > 
> > 
> > I'm working on a VScode extension that will have a template.
> 
> I haven't finished the extension yet so if I can't get it shipped this week I'll open source my progress so far.
> 
> I have been **SUPER** busy.

Still busy, swamped with work right now :(
Noticed the templates idea and that sounds interesting! I could definitely add it to the playground ^^

**Comment by qwasko1212** [comment]
@Supernova3339 @halfmoonui I have been using v1 in a couple of electron projects that were implementing different tooling for my main field - Embedded Systems. At the moment I need to build new tool and I was thinking of using the latest version of Halfmoon because it simply looks outstanding. I am not familiar with Bootstrap and that is why starting is quite hard. Is there any news regarding page builder or any examples to at least recreate https://www.gethalfmoon.com/docs/introduction page?

**Comment by Supernova3339** [comment]
> @Supernova3339 @halfmoonui I have been using v1 in a couple of electron projects that were implementing different tooling for my main field - Embedded Systems. At the moment I need to build new tool and I was thinking of using the latest version of Halfmoon because it simply looks outstanding. I am not familiar with Bootstrap and that is why starting is quite hard. Is there any news regarding page builder or any examples to at least recreate https://www.gethalfmoon.com/docs/introduction page?

I don't have control over the site, you can ask Tahmid.

**Comment by qwasko1212** [comment]
@Supernova3339 Is there any news regarding your extension plugin?

**Comment by Supernova3339** [comment]
due to work and lack of funding I have no motivation to continue
development.

**Comment by KaKi87** [comment]
@halfmoonui 😭

**Comment by Supernova3339** [comment]
It receives infrequent updates.

It is still maintained.

**Comment by KaKi87** [comment]
The last commit on this project was more than a year ago, and the author's last commit anywhere else was 6 months ago.

**Comment by ranami0822** [comment]
I miss you
